Rebalance RPL Inflation (#376)
rocket-pool/RPIPs: RPIPs/RPIP-81.md

The pDAO is increasingly asked to fund or help fund items such as support, rescue node costs, business development, Chainlink costs, Saturn 2 related work, Snapshot, and other worthy initiatives. Meanwhile, pDAO funding from megapools is not yet available at meaningful scale and it might be a long time before it becomes helpful at a meaningful level.

Before Saturn 2, 70% of RPL inflation is directed to Node Operator RPL rewards. When Saturn 2 takes effect, those Node Operator RPL rewards will go away entirely. This RPIP proposes a phase out of those rewards pre-Saturn 2 by redirecting part of the current Node Operator RPL issuance to pDAO protocol funding immediately, and to preserve that funding after Saturn 2.

## Specification

### Pre-Saturn 2
- The reward percentages **SHALL** be set using the on-chain pDAO proposal system ([RPIP-33](RPIP-33.md)) to:
  - Node Operators: 50%

>   - pDAO: 47.5%
>   - oDAO: 2.5%
>   - This may be proposed using the Smartnode CLI command: `rocketpool pdao propose rewards-percentages --node 0.5 --odao 0.025 --pdao 0.475`
>   - This on-chain proposal vote must be passed to implement the change from this vote
> - The pDAO internal allocation **SHALL** be adjusted and updated in [RPIP-10](RPIP-10.md)'s `Historical budget splits` section as follows:
>   - IMC: 30%
>   - GMC: 40%
>   - Reserve Treasury: 30%

Reviewer: - I believe changing the pDAO internal allocation will also require pDAO votes (one for IMC and one for GMC). 
- I don't think the CLI commands to start votes need to be in the specification as long as outcomes are clearly defined. 
- Treasury spends are in absolute RPL terms, so I believe it's worth being explicit about timing and what values to use.

```suggestion
- The reward percentages **SHALL** be set using the on-chain pDAO proposal system ([RPIP-33](RPIP-33.md)) to:
  - Node Operators: 50%
  - pDAO: 47.5%
  - oDAO: 2.5%
- The pDAO internal allocation **SHALL** be adjusted using the on-chain pDAO proposal system and updated in [RPIP-10](RPIP-10.md)'s `Historical budget splits` section as follows:
  - IMC: 30%
  - GMC: 40%
  - Reserve Treasury: 30%
- The on-chain votes SHALL be started concurrently and  RPL values for the pDAO allocation SHALL be based on the reward percentages vote passing.
-  All associated on-chain proposal votes must be passed to implement this proposal. 
```

Author: I agree with these changes

### Post-Saturn 2

- The RPL inflation rate **SHALL** be set to 2.5% and updated in [RPIP-46](RPIP-46.md)
- `rpl.inflation.interval.rate` **SHALL** be set to the value corresponding to 2.5% annual inflation using the same annualization convention as [RPIP-46](RPIP-46.md). The expected value is approximately `1000067606974065369`, subject to technical confirmation.

- The reward percentages **SHALL** be set using the on-chain pDAO proposal system to:
  - Node Operators: 0%
  - pDAO: 95%
  - oDAO: 5%

>   - This may be proposed using the Smartnode CLI command: `rocketpool pdao propose rewards-percentages --node 0 --odao 0.05 --pdao 0.95`

Reviewer: ```suggestion
```

Author: I agree with these changes

This RPIP attempts to balance those risks by:

- preserving 50% of RPL rewards for Node Operators before Saturn 2,
- eliminating Node Operator RPL rewards at Saturn 2 as already planned,
- keeping GMC, IMC, and Reserve Treasury spending subject to normal governance and budgeting processes, and
- leaving future governance able to reduce inflation or revise allocations if the additional funding is no longer needed.

## Governance Considerations

This RPIP modifies RPL inflation allocation and therefore requires a 75% "For" supermajority under [RPIP-25](RPIP-25.md).

Passage of this RPIP should be treated as authorization for three related governance actions:

1. A pre-Saturn 2 on-chain pDAO proposal to change RPL claim percentages to 50% Node Operators, 47.5% pDAO, and 2.5% oDAO.
2. An update to the pDAO allocation policy in [RPIP-10](RPIP-10.md) to set IMC to 30%, GMC to 40%, and Reserve Treasury to 30%.
3. An amendment to [RPIP-46](RPIP-46.md) so that the Saturn 2 `rpl.inflation.interval.rate` target is 2.5% annual inflation rather than 1.5%.

> 

Reviewer: Add the additional on-chain votes here
```suggestion
Passage of this RPIP should be treated as authorization for five related simultaneous governance actions:

1. A pre-Saturn 2 on-chain pDAO proposal to change RPL claim percentages to 50% Node Operators, 47.5% pDAO, and 2.5% oDAO.
2. A pre-Saturn 2 on-chain pDAO proposal to change recurring IMC spend to reflect the increase to total pDAO allocation and a 30% share of it.
3. A pre-Saturn 2 on-chain pDAO proposal to change recurring GMC spend to reflect the increase to total pDAO allocation and a 40% share of it.
4. An update to the pDAO allocation policy in [RPIP-10](RPIP-10.md) to set IMC to 30%, GMC to 40%, and Reserve Treasury to 30%.
5. An amendment to [RPIP-46](RPIP-46.md) so that the Saturn 2 `rpl.inflation.interval.rate` target is 2.5% annual inflation rather than 1.5%.
```

Author: Also good

---
rpip: 
title: Rebalance RPL Inflation for Protocol Funding
description: Increase the pDAO and GMC share of RPL inflation to support protocol funding.
author: Dr Doofus (@DrDoofus-MD-PhD-DDS)
discussions-to:
status: Review
type: Protocol
category: Core
created: 2026-05-21
requires: RPIP-10, RPIP-25, RPIP-33, RPIP-46, RPIP-68, RPIP-69
vote-to:
vote-date:
vote-result:
tags: [rpl-inflation, saturn-2, node-operator-percent, pdao-percent]
---

## Abstract

This RPIP proposes rebalancing RPL inflation allocation to increase protocol funding during and after the Saturn 2 transition.

Before Saturn 2, the current RPL inflation allocation sends 70% of RPL issuance to Node Operators, 27.5% to the pDAO, and 2.5% to the oDAO. This RPIP would reduce the Node Operator share to 50%, increase the pDAO share to 47.5%, and leave the oDAO share unchanged at 2.5%. This RPIP also updates the internal pDAO allocation policy to 30% IMC, 40% GMC, and 30% Reserve Treasury.

> When Saturn 2 takes effect, Node Operator share of RPL inflation is already set to go to zero ([RPIP-46](./RPIP-46.md)). Under the existing Saturn 2 plan, annual RPL inflation would thus fall from 5% to 1.5%. Under this RPIP, elimination of Node Operator RPL rewards will reduce inflation from 5% to 2.5%.

Reviewer 2: We've been doing `([RPIP-46](RPIP-46.md)`. I'm not sure if it would also work with the `./`, including on rpips.rocketpool.net, but would rather keep consistency

Author: done

## Motivation

Rocket Pool currently faces a protocol funding shortfall.

The decline in RPL price has greatly reduced the GMC’s funding capabilities. The GMC has already made multiple rounds of cuts to support, administration, business development/STAR, marketing and other recurring expenses. However, current funding is now not enough for basic operating costs and does not provide adequate room for upcoming protocol needs.

The pDAO is increasingly asked to fund or help fund items such as support, rescue node costs, business development, Chainlink costs, Saturn 2 related work, Snapshot, and other worthy initiatives. Meanwhile, pDAO funding from megapools is not yet available at meaningful scale and it might be a long time before it becomes helpful at a meaningful level.

Before Saturn 2, 70% of RPL inflation is directed to Node Operator RPL rewards. When Saturn 2 takes effect, those Node Operator RPL rewards will go away entirely. This RPIP proposes a phase out of those rewards pre-Saturn 2 by redirecting part of the current Node Operator RPL issuance to pDAO protocol funding immediately, and to preserve that funding after Saturn 2.

> ## Specification

Reviewer 2: This section currently has a lot of content beyond spec. I'd suggest moving to rationale/abstract as desired, and making a tighter spec. Here's a suggestion you can use or not as you wish:

```
## Specification

- The reward percentages **SHALL** be set using the on-chain pDAO proposal system
  - This may be proposed using the Smartnode CLI command: `rocketpool pdao propose rewards-percentages --node 0.5 --odao 0.025 --pdao 0.475`
  - This on-chain proposal vote must be passed to implement the change from this vote
- The pDAO inflation shares SHALL be adjusted, and updated in [RPIP-10](RPIP-10.md)'s `Historical budget splits` section:
  - Incentives of pDAO inflation SHALL be adjusted to 30%
  - Grants and Bounties share of pDAO inflation SHALL be adjusted to 40%
  - Reserve Treasury share of pDAO inflation SHALL be adjusted to 30%
```

Author: changed specs to be tighter. basically used this format, thanks
